frontend/src/components/layout/Navigation.css

```diff
@@ -1,3 +1,5 @@
+/* Navigation.css */
```

Comment:
you dont want comments in your production code

Reply:
done!